ArduCopter/RC_Channel.cpp

```diff
@@ -618,6 +619,22 @@ bool RC_Channel_Copter::do_aux_function(const aux_func_t ch_option, const AuxSwi
             break;
 #endif
 
+#if WEATHERVANE_ENABLED == ENABLED
```

Comment:
This should be in parent class, gated on `AP_WEATEHRVANE_ENABLED`

Reply:
The problem is that is is currently always enabled on quadplane. So we would endup having to do dummy methods on copter. Since this only ever going to be copter and plane I don't think its worth it, its not like some features that we want on all vehicles.

Reply:
Not really all that fussed.  But when we get subs or boats weather-vaning in current I *am* going to remind you of this comment....